### PR TITLE
[Training] use only one BDT template for different nuisance

### PR DESCRIPTION
**Pull request by tihsu99**

### CODE REVIEW

**Comment by efeyazgan** [comment]
Looks good to me. @gourangakole please take a look and merge. thanks.

File: BDT_application/step2_BDTapplication_condor/wrapper.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -e 
 echo "TEST FIRST" 
 echo "copy input root file"
-eos cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_input_application/YEAR/SIGNALROOT .
-eos cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_input_application/YEAR/aa.tar.gz .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_input_forApplication//YEAR/SIGNALROOT .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_input_forApplication//YEAR/aa.tar.gz .
```
Comment (gourangakole) [inline BDT_application/step2_BDTapplication_condor/wrapper.sh]: Hi @AlbertHsuNTUphys You are NOT generating the step1 input file for the application, so why did you change it? 
I would prefer to keep it as it is.

File: BDT_application/step2_BDTapplication_condor/wrapper.sh [hunk 2]
```diff
@@ -11,47 +11,7 @@ HOME=$PWD
...
-
-mkdir BDT_weights_7
-cd BDT_weights_7
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_7/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_muPtup/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_8
-cd BDT_weights_8
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_8/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_muPtdo/weights/TMVAClassification_BDTG.weights.xml .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_weight/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_0/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_central/weights/TMVAClassification_BDTG.weights.xml .
```
Comment (gourangakole) [inline BDT_application/step2_BDTapplication_condor/wrapper.sh]: same comment as above , you just use the exisiting training files.
not need to point to your new training files.

File: BDT_application/step2_BDTapplication_condor/wrapper.sh (hunk 1 above)
Comment (tihsu99) [reply BDT_application/step2_BDTapplication_condor/wrapper.sh]: Okay, let me restore it.
